src/k8s-engine/lima.ts

```diff
@@ -1314,6 +1321,17 @@ ${ commands.join('\n') }
             }
           });
 
+        if (this.#currentContainerEngine !== ContainerEngine.MOBY) {
+          // We can't install buildkitd earlier because if we were running an older version of rancher-desktop,
+          // we have to remove the kim buildkitd k8s artifacts. And we can't remove them until k8s is running.
+          if (config.checkForExistingKimBuilder) {
+            await getImageProcessor(this.#currentContainerEngine, this).removeKimBuilder();
```

Comment:
We made a client for _Waiting for services_; it would be nice to be able to reuse it.

Reply:
Thanks, found a way to use it.